ext/authx/Civi/Authx/Authenticator.php

```diff
@@ -250,6 +250,36 @@ protected function checkPolicy(AuthenticatorTarget $tgt) {
     }
   }
 
+  /**
+   * Check for a pre-existing login
+   *
+   * @param AuthenticatorTarget $tgt
+   * @return bool
+   */
+  protected function checkAlreadyLoggedIn(AuthenticatorTarget $tgt) {
+    $existingContact = \CRM_Core_Session::getLoggedInContactID();
+    $existingUser = $this->authxUf->getCurrentUserId();
+
+    if (!$existingContact && !$existingUser) {
```

Comment:
If I put an OR here then then it works for me

Reply:
ie what works is a situation where coworker is running intiates a login from NULL to something